test/integration/taxon_browsing_test.rb

```diff
@@ -141,7 +141,7 @@ def given_there_is_a_taxon_with_grandchildren
 
     @child_taxons =
       funding_and_finance_for_students_taxon['links']['child_taxons']
-    assert @child_taxons.length > 0
+    assert @child_taxons.length.positive?
```

Comment:
Doesn't this already happen in the previous commit? Or were there two linting errors that happened to end up on line 144?